### DISCUSSION TITLE
stopwatch

### DISCUSSION THREAD

**Discussion by chipset2011** [zepp-health/zeppos-forum#353] [General]
Dear developers, please post the stopwatch project, I want to correct it for myself, for example, to make the start/stop control by the button, I asked to add these functions a long time ago, but they were not added, I want to correct the application for greater convenience, at least for personal use

**Comment by maxiges** [comment]
This is probably not possible when official applications are not open source, but you can build applications yourself

https://docs.zepp.com/docs/intro/
